### PR TITLE
First try of Kotlin documenting

### PR DESCRIPTION
**Pull request by kiranofans**
- Added KDoc to ProviderUtils.kt, SharedPrefs.kt, ArticlesEditedRecyclerAdapter.kt, and CampaignListRecyclerAdapter.kt

### CODE REVIEW

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/adapters/ArticlesEditedRecyclerAdapter.kt [hunk 1]
```diff
@@ -10,7 +10,11 @@ import org.wikiedufoundation.wikiedudashboard.R
 import org.wikiedufoundation.wikiedudashboard.ui.coursedetail.articlesedited.data.Article
 import java.util.*
 
-class ArticlesEditedRecyclerAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+/**
+ * A RecyclerView adapter for edited articles
+ * @property context Context
+***/
+class ArticlesEditedRecyclerAdapter(private val context: Context) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```
Comment (ujjwalagrawal17) [inline app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/adapters/ArticlesEditedRecyclerAdapter.kt]: @kiranofans can you revert this change? You have added an extra variable context. If you pull new code from main repo master branch I think it should automatically revert this.

Comment (kiranofans) [reply app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/adapters/ArticlesEditedRecyclerAdapter.kt]: Hum, it's weird. I didn't add the context variable...

So I will try to revert it

Comment (kiranofans) [reply app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/adapters/ArticlesEditedRecyclerAdapter.kt]: I just checked, on the Master branch, the ArticlesRecyclerAdapter looks like this 
```
class ArticlesEditedRecyclerAdapter(private val context: Context) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

**Comment by kiranofans** [comment]
Committed, please check.  
Thanks
